src/connectors/relevant-sort/connectRelevantSort.ts

```diff
@@ -109,17 +123,15 @@ const connectRelevantSort: RelevantSortConnector = function connectRelevantSort(
       getWidgetSearchParameters(state, { uiState }) {
         return state.setQueryParameter(
           'relevancyStrictness',
-          uiState.relevantSort?.relevancyStrictness ?? state.relevancyStrictness
+          uiState.relevantSort ?? state.relevancyStrictness
         );
       },
 
       getWidgetUiState(uiState, { searchParameters }) {
         return {
           ...uiState,
-          relevantSort: {
-            ...uiState.relevantSort,
-            relevancyStrictness: searchParameters.relevancyStrictness,
-          },
+          relevantSort:
+            searchParameters.relevancyStrictness || uiState.relevantSort,
```

Comment:
uiState change is technically a breaking change, but small enough to just mention in changelog

Reply:
`|| uiState.relevantSort`

Does this mean we can now set the `relevantSort` from the frontend?